librecores-ci/README.md

```diff
@@ -14,3 +14,17 @@ This is a base image which includes common EDA tools.
 * [cocotb](https://github.com/potentialventures/cocotb)
 * [pytest](https://docs.pytest.org/en/latest/)
 * [tap.py](https://pypi.org/project/tap.py/)
+
+### Quick start 
```

Comment:
Maybe it worth adding a "Links" session and referencing your blogposts there

Reply:
Done